python/podio_class_generator.py

```diff
@@ -222,15 +229,40 @@ def _process_datatype(self, name, definition):
     self._fill_templates('Obj', datatype)
     self._fill_templates('Collection', datatype)
     self._fill_templates('CollectionData', datatype)
+    self._fill_templates('MutableStruct', datatype)
+    self._fill_templates('Constructor', datatype)
 
     if 'SIO' in self.io_handlers:
       self._fill_templates('SIOBlock', datatype)
 
+  def _preprocess_for_julia(self, datatype):
+    includes_jl = set()
+    for relation in datatype['OneToManyRelations'] + datatype['VectorMembers'] + datatype['OneToOneRelations']:
+      if self._needs_include(relation):
+        if not relation.is_builtin:
+          if relation.full_type == datatype['class'].full_type:
+            includes_jl.add(self._build_include(datatype['class'].bare_type, julia=True, is_struct=True))
+          else:
+            includes_jl.add(self._build_include(relation.bare_type, julia=True, is_struct=True))
+          # if datatype['class'].bare_type != relation.bare_type:
+          #   includes_jl.add(self._build_include(relation.bare_type + 'Collection', julia=True))
+    try:
+      includes_jl.remove(self._build_include(datatype['class'].bare_type, julia=True, is_struct=True))
+    except KeyError:
+      pass
+    datatype['includes_jl']['constructor'].update((includes_jl))
+
+  def _get_params(self, datatype):
```

Comment:
```suggestion
  def _get_julia_params(self, datatype):
```
Just to make it clear that this is necessary for Julia.

Could you also put a docstring here that very briefly explains what this function does and why it is necessary?

Reply:
I have already included the docstring.

Reply:
Ah right, Looks like I was just commenting while you were fixing the pre-commit issue for the docstring.